meta-gnome/recipes-support/onboard/onboard/0002-onboard-onhover-seg-fault-fix.patch

```diff
@@ -0,0 +1,52 @@
+From: 1be95325d320122efd5dedf7437839cfcca01f7a
+From: https://github.com/void-linux/void-packages/commit/1be95325d320122efd5dedf7437839cfcca01f7a
+Date: Mon, 23 Sep 2024
+Subject: [PATCH] onboard: fix segfault when hovering over the keyboard
+
+Currently, if the mouse pointer is hovered over the Onboard keyboard, the application crashes with a segmentation fault.
+This is because the new_device_event function is not acquiring the GIL before creating a new object. This patch fixes
+the issue by acquiring the GIL before creating a new object. It also acquires the GIL before queueing the event.
+This patch is taken from the fix provided in the following link:
+https://github.com/void-linux/void-packages/commit/1be95325d320122efd5dedf7437839cfcca01f7a
+
+Signed-off by: Pratheeksha S N <pratheeksha.s.n@ni.com>
+
+Upstream-Status: Not Submitted [This is being added to the meta-nilrt layer for now. If it is accepted upstream, it will be removed from here.]
```

Comment:
This is not accurate, please fix.

Refer https://docs.yoctoproject.org/contributor-guide/recipe-style-guide.html#patch-upstream-status - I think it can be classified as "Inactive-Upstream", and the reason to `[lastcommit: when (and/or) lastrelease: when]`.

I know we mention `Not Submitted` as one of the reasons in https://github.com/ni/meta-nilrt/blob/nilrt/master/next/docs/styleguide.md#L133 but haven't found mention of it upstream.

Reply:
I have fixed this now.